CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ vmware.vmware_rest Release Notes
 
 .. contents:: Topics
 
+v3.0.1
+===========
+
```

Comment:
Can you add release_summary? (Refer: https://github.com/ansible-collections/vmware.vmware_rest/pull/481/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cL29)